lib/actions/comment.js

```diff
@@ -1,9 +1,9 @@
 const { Action } = require('./action')
 const populateTemplate = require('./handlebars/populateTemplate')
 
-const createComment = async (context, number, body) => {
+const createComment = async (context, issueNumber, body) => {
   return context.github.issues.createComment(
-    context.repo({ number, body })
+    context.repo({ issue_number: issueNumber, body })
```

Comment:
`number` was deprecated in favor of `issue_number`.

Same goes for the `number` -> `pull_number` changes in other files.